orchestrator/nodes/http/index.js

```diff
@@ -80,29 +80,34 @@ class DataHandler extends dojot.DataHandlerBase {
     handleMessage(config, message, callback) {
 
         var nodeUrl = config.url;
-        var isTemplatedUrl = (nodeUrl || "").indexOf("{{") != -1;
+        var isTemplatedUrl = (nodeUrl || "").indexOf("{{") !== -1;
         var nodeMethod = config.method || "GET";
         var ret = config.ret || "txt";
-        var reqTimeout = 120000
+        var reqTimeout = 120000;
         var url = nodeUrl || message.url;
         var requestPayload = this._get(config.body, message);
 
+
+        // Pre-process URL.
+
+        // First, resolve URL if it uses a mustache string.
         if (isTemplatedUrl) {
-            url = mustache.render(nodeUrl, messsage);
+            url = mustache.render(nodeUrl, message);
         }
 
         if (!url) {
-            callback("httpin.errors.no-url", [])
+            callback("httpin.errors.no-url", []);
             return;
         }
 
-        // url must start http:// or https:// so assume http:// if not set
+        // Then, check whether it is correctly set - starts with http:// or https://
         if (url.indexOf("://") !== -1 && url.indexOf("http") !== 0) {
```

Comment:
I think this condition is not correct. Something like:
url = "http:/blabla"
it gives us a false result.